server/data/assessmentClient.test.ts

```diff
@@ -33,15 +34,23 @@ describe('AssessmentClient', () => {
   describe('all', () => {
     const assessmentSummaries = assessmentSummaryFactory.buildList(5)
 
-    it('should get all assessments for the current user', async () => {
-      fakeApprovedPremisesApi
-        .get(paths.assessments.index({}))
-        .matchHeader('authorization', `Bearer ${callConfig.token}`)
-        .reply(200, assessmentSummaries)
-
-      const output = await assessmentClient.all()
-      expect(output).toEqual(assessmentSummaries)
-    })
+    it.each([
+      ['unallocated', 'unallocated' as AssessmentStatus],
```

Comment:
possibly not, but would casting these `as const` allow TS to infer them as `AssessmentStatus`es?